backend/composer/admin.py

```diff
@@ -42,7 +42,7 @@ class ProvenanceInline(admin.StackedInline):
     model = Provenance
     extra = 1
 
-class SynonymInline(admin.StackedInline):
+class SynonymeInline(admin.StackedInline):
```

Comment:
@D-GopalKrishna this is not worded correctly please change Synonyme into Synonym